Casks/v/visual-studio-code.rb

```diff
@@ -1,23 +1,37 @@
 cask "visual-studio-code" do
   arch arm: "darwin-arm64", intel: "darwin"
 
-  version "1.98.0"
-  sha256 arm:   "c794330a7e7b96a6f31d69c67b01c52f4d3a7574fc56d50d24e578b06495e72d",
-         intel: "0a5cebab8e2939592a631d162bec15589cfe573e2ebfb5cd466107ba1d0151d9"
+  on_catalina do
```

Comment:
```suggestion
  on_catalina :or_older do
```

This ensures that the cask definition still loads on older software, but wouldn't be installed because of the `depends_on macos: ">= :catalina"` further down in the file.